lib/vm/opFns.js

```diff
@@ -91,7 +90,16 @@ module.exports = {
     if (exponent.isZero()) {
```

Comment:
Another shortcut like this could be `if (base.isZero()) return new BN(0)`

Reply:
Added this after the gas calculation since this is gas-cost relevant, should work, let's wait for the tests to be sure.